setup.cfg

```diff
@@ -18,6 +18,7 @@ addopts =
   -p no:logging
   --color=yes
   --doctest-glob=*.rst
+  --doctest-modules
```

Comment:
ah, this needs to be removed, or we need to fix a bug in an upstream library.
I'll look into that tomorrow.

Reply:
Created https://github.com/timofurrer/ramlient/issues/51